### PR TITLE
500エラーでるようにした.カート追加ボタン押下時にログインしてないとカート一覧をみれないのを修正

### PR DESCRIPTION
**Pull request by tkeshun**

### CODE REVIEW

File: src/main/java/com/example/controller/ConfirmOrderController.java [hunk 1]
```diff
@@ -31,14 +31,14 @@ public class ConfirmOrderController {
 	@GetMapping("/to-confirm-order")
 	public String toConfirmOrder(Integer orderId, Model model) {
 		Order order = confirmOrderService.showOrder(orderId);
-		
+
 		System.out.println(order);
-		for(OrderItem orderItem : order.getOrderItemList()) {
+		for (OrderItem orderItem : order.getOrderItemList()) {
```
Comment (tkeshun) [inline src/main/java/com/example/controller/ConfirmOrderController.java:36]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (tkeshun) [inline src/main/java/com/example/controller/ConfirmOrderController.java:36]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'for' has incorrect indentation level 16, expected level should be 4.

File: src/main/java/com/example/common/GlobalExceptionHandler.java [hunk 2]
```diff
@@ -2,31 +2,27 @@
...
 	private static final Logger logger = LoggerFactory.getLogger(GlobalExceptionHandler.class);
 
-	/* (non-Javadoc)
-	 * @see org.springframework.web.servlet.HandlerExceptionResolver#resolveException(javax.servlet.http.HttpServletRequest, javax.servlet.http.HttpServletResponse, java.lang.Object, java.lang.Exception)
-	 */
-	@Override
-	public ModelAndView resolveException(HttpServletRequest request, HttpServletResponse response, Object obj, Exception e) {
-		logger.error("システムエラーが発生しました！", e);
-		return null;
+	@ExceptionHandler(Throwable.class)
```
Comment (tkeshun) [inline src/main/java/com/example/common/GlobalExceptionHandler.java:20]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (tkeshun) [inline src/main/java/com/example/common/GlobalExceptionHandler.java:20]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def modifier' has incorrect indentation level 8, expected level should be 2.

Comment (tkeshun) [inline src/main/java/com/example/common/GlobalExceptionHandler.java:20]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.javadoc.MissingJavadocMethodCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Missing a Javadoc comment.

File: src/main/java/com/example/common/GlobalExceptionHandler.java [hunk 3]
```diff
@@ -2,31 +2,27 @@
...
+	public String handleConnectionError(Exception e) {
```
Comment (tkeshun) [inline src/main/java/com/example/common/GlobalExceptionHandler.java:21]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

File: src/main/java/com/example/common/GlobalExceptionHandler.java [hunk 4]
```diff
@@ -2,31 +2,27 @@
...
+
+		logger.error(e.getMessage(), e);
```
Comment (tkeshun) [inline src/main/java/com/example/common/GlobalExceptionHandler.java:23]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (tkeshun) [inline src/main/java/com/example/common/GlobalExceptionHandler.java:23]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def' child has incorrect indentation level 16, expected level should be 4.

File: src/main/java/com/example/common/GlobalExceptionHandler.java [hunk 5]
```diff
@@ -2,31 +2,27 @@
...
+
+		return "error/500";
```
Comment (tkeshun) [inline src/main/java/com/example/common/GlobalExceptionHandler.java:25]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (tkeshun) [inline src/main/java/com/example/common/GlobalExceptionHandler.java:25]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def' child has incorrect indentation level 16, expected level should be 4.

File: src/main/java/com/example/common/SecurityConfig.java [hunk 6]
```diff
@@ -42,7 +42,7 @@ SecurityFilterChain filterChain(HttpSecurity http) throws Exception {
 		http.authorizeHttpRequests(ahr -> ahr
 				.requestMatchers("/login-user/to-login", "/register-user/to-register", "/login-user/login-user",
 						"/register-user/register-user", "/show-item/show-item-list", "/shopping-cart/show-cart",
-						"/show-detail/show-item-detail")
+						"/shopping-cart/add-item", "/show-detail/show-item-detail")
```
Comment (tkeshun) [inline src/main/java/com/example/common/SecurityConfig.java:45]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.sizes.LineLengthCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line is longer than 100 characters (found 107).

Comment (tkeshun) [inline src/main/java/com/example/common/SecurityConfig.java:45]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

**Review by yousuke-murayama — APPROVED** [review]
問題ないと思います。

File: src/main/java/com/example/common/SecurityConfig.java [hunk 7]
```diff
@@ -42,7 +42,7 @@ SecurityFilterChain filterChain(HttpSecurity http) throws Exception {
 		http.authorizeHttpRequests(ahr -> ahr
 				.requestMatchers("/login-user/to-login", "/register-user/to-register", "/login-user/login-user",
 						"/register-user/register-user", "/show-item/show-item-list", "/shopping-cart/show-cart",
-						"/show-detail/show-item-detail", "/show-top-page/top")
+						"/shopping-cart/add-item", 	"/show-detail/show-item-detail", "/show-top-page/top")
```
Comment (tkeshun) [inline src/main/java/com/example/common/SecurityConfig.java:45]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.sizes.LineLengthCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line is longer than 100 characters (found 134).

Comment (tkeshun) [inline src/main/java/com/example/common/SecurityConfig.java:45]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

**Pull request merged**